cpp/cmake/thirdparty/get_nvcomp.cmake

```diff
@@ -32,9 +33,8 @@ function(find_and_configure_nvcomp VERSION)
   if(TARGET nvcomp AND PER_THREAD_DEFAULT_STREAM)
     target_compile_definitions(nvcomp PRIVATE CUDA_API_PER_THREAD_DEFAULT_STREAM)
   endif()
-
 endfunction()
 
-set(CUDF_MIN_VERSION_nvCOMP 2.1.0)
-
-find_and_configure_nvcomp(${CUDF_MIN_VERSION_nvCOMP})
+set(CUDF_MIN_VERSION_nvCOMP 2.2)
+set(CUDF_MAX_VERSION_nvCOMP 2.3)
```

Comment:
I assume that removing the patch component was intentional here? We aren't dependent on any particular bug fixes in 2.2.x?

Reply:
Dropping of the patch value was a mistake, I have restored the proper patch values.